docs/bigquery/snippets.py

```diff
@@ -2297,11 +2297,31 @@ def test_client_list_jobs(client):
     # [START bigquery_list_jobs]
     # from google.cloud import bigquery
     # client = bigquery.Client(project='my_project')
+    import datetime
 
     # List the 10 most recent jobs in reverse chronological order.
     # Omit the max_results parameter to list jobs from the past 6 months.
+    print("Last 10 jobs:")
     for job in client.list_jobs(max_results=10):  # API request(s)
         print(job.job_id)
+
+    # The following are examples of additional optional parameters:
+
+    # Use min_creation_time and/or max_creation_time to specify a time window.
+    print("Jobs from the last two days:")
+    two_days_ago = datetime.datetime.utcnow() - datetime.timedelta(days=2)
```

Comment:
Thanks for the update. I wonder if 2 days is too many for the CircleCI project, since it runs a lot of jobs. Maybe cut it to the last 10 minutes?

Reply:
Trimming the number of jobs listed SGTM.

Reply:
Yeah, 10 minutes should be sufficient for this case.